indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -133,3 +136,12 @@ def _process(self):
             logger.info("Registrations for {} scheduled by {}".format(self.regform, session.user))
             return jsonify_data(flash=False)
         return jsonify_template('events/registration/management/regform_schedule.html', form=form)
+
+
+class RHRegistrationFormModify(RHManageRegFormBase):
+    """Modify the form of a registration form"""
```

Comment:
![](http://static4.fjcdn.com/thumbnails/comments/So+she+was+a+boy+pretending+to+be+a+girl+_eb6d5a2ac05735ae8399c11ae47f5652.png)